ui/app/components/breadcrumbs/default.js

```diff
@@ -0,0 +1,18 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import KeyboardShortcutModifier from 'nomad-ui/modifiers/keyboard-shortcut';
+import { inject as service } from '@ember/service';
+
+export default class BreadcrumbsTemplate extends Component {
+  @service router;
+
+  @action
+  traverseUpALevel(args) {
+    const [path, ...rest] = args;
+    this.router.transitionTo(path, ...rest);
+  }
+
+  get maybeKeyboardShortcut() {
+    return this.args.isOneCrumbUp() ? KeyboardShortcutModifier : null;
+  }
```

Comment:
`nit` `suggestion`: This has a fixed value.

The function definition is:

```
  isOneCrumbUp(iter = 0, totalNum = 0) {
    return iter === totalNum - 2;
  }
```

No arguments are passed to this function, thus it will always be `false`.

Reply:
This is perhaps made more complicated by virtue of the curried function having the same name as the applied function, but this refers to the isOneCrumbUp defined in the component here: https://github.com/hashicorp/nomad/blob/963cc40f0807c0409f2f45ca82332b2ee1a93222/ui/app/templates/components/app-breadcrumbs.hbs#L4